specification/redis/resource-manager/Microsoft.Cache/stable/2022-05-01/redis.json

```diff
@@ -1898,7 +1898,7 @@
             },
             "rdb-backup-frequency": {
               "type": "string",
-              "description": "Specifies the frequency for creating rdb backup"
+              "description": "Specifies the frequency for creating rdb backup. Valid values: (15, 30, 60, 360, 720, 1440)"
```

Comment:
This would be 'in minutes' right? That might be clearer.

Reply:
I also notice there don't seem to be any examples covering this property today.

Reply:
Yes, this is in minutes and I have made it clearer now.

Reply:
@gorandom Thanks!